webapp/src/components/SheetTabBar/SheetRenameDialog.tsx

```diff
@@ -13,35 +14,22 @@ interface SheetRenameDialogProps {
 const SheetRenameDialog = (properties: SheetRenameDialogProps) => {
   const { t } = useTranslation();
   const [name, setName] = useState(properties.defaultName);
+  const crossRef = useRef<HTMLDivElement>(null);
   const handleClose = () => {
     properties.onClose();
   };
   return (
     <Dialog open={properties.open} onClose={properties.onClose}>
       <StyledDialogTitle>
         {t("sheet_rename.title")}
-        <Cross onClick={handleClose} onKeyDown={() => {}}>
-          <svg
-            width="16"
-            height="16"
-            viewBox="0 0 16 16"
-            fill="none"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <title>Close</title>
-            <path
-              d="M12 4.5L4 12.5"
-              stroke="#333333"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-            <path
-              d="M4 4.5L12 12.5"
-              stroke="#333333"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </svg>
+        <Cross
+          onClick={handleClose}
+          title="Close Dialog"
+          ref={crossRef}
+          tabIndex={0}
+          onKeyDown={() => {}}
```

Comment:
The `onKeyDown` event is currently not doing anything.

It's extra code I added some time ago. Also, `ref` and `tabIndex` aren’t doing anything right now:  
[MDN Documentation on `tabIndex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

In that documentation, you can read about the `-1` value. It’s used so the element can receive keyboard focus. This solves two issues:

1. **Focus in a widget with no other focusable tags**  
   When the dialog opens, it grabs the focus. Without a focusable element, the focus would remain on the workbook. This is useful for capturing the `ESC` keypress event, for example.

2. **Capturing the `Enter` keypress when the X is focused**  
   We can use the `onKeyDown` event to catch the `Enter` keypress when the close button (the X) has focus.

The update dialog behaves incorrectly. If you open it and start pressing the tab key, you’ll see that the focus eventually moves to the workbook. That doesn’t happen with the other dialogs, such as the rename or delete workbook dialogs.

Reply:
Oh I see. Thanks for the explanation. So it would make sense to also add the tabIndex to the X and 'Save' buttons, in all dialogs, as well?

> The update dialog behaves incorrectly. 
You mean the "Import' dialog?

Reply:
> You mean the "Import' dialog?

Yes, sorry.

> So it would make sense to also add the tabIndex to the X and 'Save' buttons, in all dialogs, as well?

So elements like inputs and texareas are focusable (I'm not sure if that is a word) and you don't need a tabIndex on them

Reply:
You don't need tabIndex and onKeydown here unless you are planning on using them.

One neat use of it would be to exit when pressing enter while the X is focused

Reply:
Should be fixed now.

Reply:
You still don't need the "onKeyDown"

Let's tackle this and other issues in a new PR.